scalding-core/src/main/scala/com/twitter/scalding/StringUtility.scala

```diff
@@ -0,0 +1,39 @@
+package com.twitter.scalding
+
+object StringUtility {
```

Comment:
do we have some benchmarks to show this is faster? I'm concerned that there are a few things in this code that look slow (List ++, foldRight on the string).